front-end/package.json

```diff
@@ -32,6 +32,8 @@
     "jest-canvas-mock": "^2.3.1",
     "jest-esm-transformer": "^1.0.0",
     "lerna": "^4.0.0",
+    "less": "^4.1.3",
+    "less-loader": "^7.3.0",
```

Comment:
Why are `less` and `less-loader` required for this PR?
Try not to import new libraries unless it's necessary.